docs/clients/multi_turn_angle_control.rst

```diff
@@ -164,21 +164,21 @@ Type ID 59 | Multi-turn Angle Controller
 +--------+---------------------------------+----------------+-----------+-------------------+--------------------------------------------------------------------------------------------------+
 | 20     | trajectory_angular_velocity     | get, set       | float     | rad/s             | Final velocity of the trajectory. Defaults to 0.                                                 |
 +--------+---------------------------------+----------------+-----------+-------------------+--------------------------------------------------------------------------------------------------+
-| 21     | trajectory_angular_acceleration | get, set       | float     | .. math:: rad/s^2 | Final acceleration of the trajectory. Defaults to 0.                                             |
+| 21     | trajectory_angular_acceleration | get, set       | float     | rad/s^2           | Final acceleration of the trajectory. Defaults to 0.                                             |
```

Comment:
If you look at the docs as they are now, you'll see that the entries with the ..math directive look a lot nicer, but keep the whole table from looking uniform. I want to go and change everything to use it, but until we can do that, we should take it out here so we can have everything looking uniform.